CONTRIBUTING.md

```diff
@@ -107,6 +107,26 @@ Here's how to make a one-off code change. (If you're working on a larger feature
 
 7. **Celebrate.** Congratulations, you have contributed to Oppia!
 
+### Notes
+
+We do not use author tags in files, since they tend to be inaccurate or become stale when the author is no longer a regular contributor. However, you can still find the author of a particular change in a file by running the command:
+
+You can find the author of a particular change in a file as follows: 
```

Comment:
Um -- delete this line too? (Otherwise, what you've written doesn't make sense...)